### PR TITLE
In 135438679 add address block for person

### PR DESCRIPTION
**Pull request by sivaOSI**

### CODE REVIEW

File: spec/features/person/address/edit_address_spec.rb [hunk 1]
```diff
@@ -21,8 +21,8 @@
   before do
     stub_request(:get, api_person_path(person.id))
       .and_return(body: person.to_json,
-    status: 200,
-    headers: { 'Content-Type' => 'application/json' })
+                  status: 200,
+                  headers: { 'Content-Type' => 'application/json' })
   end
```
Comment (pemaOSI) [inline spec/features/person/address/edit_address_spec.rb]: why do I see dark green rectangle?

File: app/assets/javascripts/components/people/AddressEditView.jsx [hunk 2]
```diff
@@ -0,0 +1,89 @@
+import Immutable from 'immutable'
+import React from 'react'
+import AddressField from 'components/common/AddressField'
+
+export class AddressEditView extends React.Component {
```
Comment (NateOSI) [inline app/assets/javascripts/components/people/AddressEditView.jsx]: This should probably be named AddressesEditView as it renders more than one address

File: app/assets/javascripts/components/common/AddressField.jsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
+import ADDRESS_TYPE from 'AddressType'
+import InputField from 'components/common/InputField'
+import React from 'react'
+import SelectField from 'components/common/SelectField'
+import US_STATE from 'USState'
+
+const AddressField = ({streetAddress, city, state, zip, type, onChange}) => (
```
Comment (NateOSI) [inline app/assets/javascripts/components/common/AddressField.jsx]: This should be AddressEditView

File: app/assets/javascripts/components/common/Autocompleter.jsx [hunk 4]
```diff
@@ -54,7 +54,9 @@ export default class Autocompleter extends React.Component {
   }
 
   renderSuggestion(suggestion) {
-    const {first_name, last_name, gender, date_of_birth, ssn, address} = suggestion
+    const {first_name, last_name, gender, date_of_birth, ssn, addresses} = suggestion
```
Comment (NateOSI) [inline app/assets/javascripts/components/common/Autocompleter.jsx]: Allow person auto completer to handle multiple addresses

**Pull request merged**